soh/src/overlays/actors/ovl_Bg_Spot06_Objects/z_bg_spot06_objects.c

```diff
@@ -6,6 +6,7 @@
 
 #include "z_bg_spot06_objects.h"
 #include "objects/object_spot06_objects/object_spot06_objects.h"
+#include "../../../../soh/Enhancements/custom-message/CustomMessageTypes.h"
```

Comment:
this probably would work as
```suggestion
#include "soh/Enhancements/custom-message/CustomMessageTypes.h"
```